why3/src/declaration.rs

```diff
@@ -16,6 +16,10 @@ use serde::{Deserialize, Serialize};
 pub struct Module {
     pub name: Ident,
     pub decls: Vec<Decl>,
+    pub attrs: Vec<Attribute>,
+    // Meta data stored in comments
+    // Stores a pretty representation of the impl
+    pub meta: Option<String>,
```

Comment:
If we start using this more often we may just want to add `Comment` nodes to our ast.